modules/tools/rode/rode-values.tpl

```diff
@@ -1,30 +1,21 @@
-grafeas:
-  certificates:
-    name: ${grafeas_cert}
+ingress:
+  enabled: ${ingress_enabled}
+  hosts:
+    - host: ${ingress_hostname}
+      paths:
+        - /
+  annotations:
+    ${indent(4, yamlencode(ingress_annotations)) ~}
 
-certificates:
-  name: ${rode_cert}
+auth:
+  oidc:
+    ${indent(4, yamlencode(oidc_config)) ~}
 
-rbac:
-  serviceAccountAnnotations:
-    eks.amazonaws.com/role-arn: ${iam_arn}
+rode-ui:
+  enabled: false
 
-localstack:
-  enabled: ${localstack_enabled}
 
-resources:
```

Comment:
`rode-ui` doesn't specify defaults, but the others do. Do you think it would be better to set that as a default in the chart, or set it here? I'd lean towards setting it in the chart; I wouldn't expect everybody consuming the chart to figure out limits on their own?

Reply:
Interesting, I didn't realize we had defaults in the chart. Usually charts leave resources unset, but if we're already specifying them then I agree it makes sense to set them for rode-ui in the chart as well.